src/ipahealthcheck/system/filesystemspace.py

```diff
@@ -2,13 +2,29 @@
 # Copyright (C) 2019 FreeIPA Contributors see COPYING for license
 #
 
+import os
 import shutil
 
 from ipahealthcheck.system.plugin import SystemPlugin, registry
 from ipahealthcheck.core.plugin import duration, Result
 from ipahealthcheck.core import constants
 
 
+def in_container():
+    """Determine if we're running in a container."""
+    with open('/proc/1/sched', 'r') as sched:
+        data = sched.readline()
+
+    checks = [
+        data.split()[0] not in ('systemd', 'init',),
+        os.path.exists('/.dockerenv'),
+        os.path.exists('/.dockerinit'),
```

Comment:
/.dockerenv and /.dockerinit are deprecated and will probably go away in the future. I'd rather not depend on these.

Reply:
It might be better to check if the first line of /proc/1/cgroup does not contain "docker".